src/particles/elements/ChrUniformAcc.H

```diff
@@ -0,0 +1,211 @@
+/* Copyright 2022-2023 The Regents of the University of California, through Lawrence
+ *           Berkeley National Laboratory (subject to receipt of any required
+ *           approvals from the U.S. Dept. of Energy). All rights reserved.
+ *
+ * This file is part of ImpactX.
+ *
+ * Authors: Chad Mitchell, Axel Huebl
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef IMPACTX_CHRACC_H
+#define IMPACTX_CHRACC_H
+
+#include "particles/ImpactXParticleContainer.H"
+#include "mixin/beamoptic.H"
+#include "mixin/thick.H"
+#include "mixin/nofinalize.H"
+
+#include <AMReX_Extension.H>
+#include <AMReX_REAL.H>
+
+#include <cmath>
+
+
+namespace impactx
+{
+    struct ChrAcc
+    : public elements::BeamOptic<ChrAcc>,
+      public elements::Thick,
+      public elements::NoFinalize
+    {
+        static constexpr auto name = "ChrAcc";
+        using PType = ImpactXParticleContainer::ParticleType;
+
+        /** Acceleration in a uniform field Ez, with a uniform
+         *  solenoidal field Bz.  The Hamiltonian is expanded to
+         *  second order in the transverse variables (x,px,y,py),
+         *  while the exact pt dependence is retained.
```

Comment:
Separating title from body for Doxygen:
```suggestion
        /** Acceleration in a uniform field Ez, with a uniform solenoidal field Bz.
         *
         * The Hamiltonian is expanded through second order in the
         * transverse variables (x,px,y,py), with the exact pt dependence retained.
```